recipes/geant4-data-channeling/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set dataset_name = "CHANNELING" %}
+{% set version = "1.0.0" %}
+{% set version_canonical_geant4 = "1.0" %}
+{% set env_var = "G4CHANNELINGDATA" %}
+{% set summary = "GEANT4 data files for the simulations of channeling trajectories" %}
+
+{% set name = "geant4-data-"+dataset_name.lower() %}
+{% set install_dir = "share/Geant4/data/"+dataset_name+version_canonical_geant4 %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  url: https://geant4-data.web.cern.ch/geant4-data/datasets/G4{{ dataset_name }}.{{ version_canonical_geant4 }}.tar.gz
+  sha256: 203e3c69984ca09acd181a1d31a9b0efafad4bc12e6c608f0b05e695120d67f2
+  folder: dataset
+
+build:
+  number: 0
+  noarch: generic
+  script:
+    - mkdir -p ${PREFIX}/{{ install_dir }}
+    - cp -r dataset/* ${PREFIX}/{{ install_dir }}
+    - ${RECIPE_DIR}/create_activation_scripts.sh {{ name }} {{ env_var }} {{ install_dir }}
+
+# Conda-forge is unable to build packages that have no dependencies so add
+# `make` as a dummy build dependency
+requirements:
+  build:
+    - make
+
+test:
+  commands:
+    - '[ "${{ env_var }}" == "$CONDA_PREFIX/{{ install_dir }}" ]'
+
+about:
+  home: https://geant4.web.cern.ch/
+  summary: {{ summary }}
+  license: OTHER
```

Actually, we still need to attach the correct license. OTHER is a bit unspecific.
```suggestion
  license: LicenseRef-Geant4
```